meshroom/nodes/aliceVision/CameraInit.py

```diff
@@ -162,6 +162,13 @@ class CameraInit(desc.CommandLineNode):
             value=os.environ.get('ALICEVISION_SENSOR_DB', ''),
             uid=[],
         ),
+        desc.File(
+            name='colorProfileDatabase',
+            label='Color Profile Database',
+            description='''Color Profile database directory path.''',
+            value=os.environ.get('ALICEVISION_COLOR_PROFILE_DB', ''),
```

Comment:
We need to rebase on develop and change it to:
```suggestion
            value='$ALICEVISION_COLOR_PROFILE_DB',
```

Reply:
To follow the changes introduced in https://github.com/alicevision/meshroom/pull/1727/commits/7ec65d828cec87224841eb8786c9cb4c63f7d201